tidb-cloud/terraform-use-import-resource.md

```diff
@@ -0,0 +1,235 @@
+       csv_format                    = {
+           separator = ";"
+       }
+       data_format                   = "CSV"
+       elapsed_time_seconds          = 49
+       file_name                     = "./t.csv"
+       id                            = "781074"
+       new_file_name                 = "2023-02-06T05:39:42Z-t.csv"
+       pending_tables                = 0
+       post_import_completed_percent = 100
+       processed_source_data_size    = "31"
+       project_id                    = "1372813089191151295"
+       status                        = "COMPLETED"
+       target_table                  = {
+           database = "test"
+           table  = "import_test"
+       }
+       total_files                   = 0
+       total_size                    = "31"
+       total_tables_count            = 1
+       type                          = "LOCAL"
+   }
+   ```
+
+   When the status turns to `COMPLETED`, it indicates that the import task is finished.
+
+6. Check the import data with MySQL CLI:
+
+   ```
+   mysql> select * from test.import_test;
+   +------+-------+------+
+   | id   | name  | age  |
+   +------+-------+------+
+   |    1 | Alice |   20 |
+   |    2 | Bob   |   30 |
+   +------+-------+------+
+   2 rows in set (0.24 sec)
+   ```
+
+### Update an import task
+
+Import tasks cannot be updated.
+
+### Delete an import task
+
+For Terraform, deleting an import task means canceling the corresponding import resource. 
+
+you cannot cancel a `COMPLETED` import task. For example, if you cancel an import task that is completed, you will see get the `Delete Error` as follows:
+
+```
+$ terraform destroy
+...
+Plan: 0 to add, 0 to change, 1 to destroy.
+
+Do you really want to destroy all resources?
+  Terraform will destroy all your managed infrastructure, as shown above.
+  There is no undo. Only 'yes' will be accepted to confirm.
+
+  Enter a value: yes
+
+tidbcloud_import.example_local: Destroying... [id=781074]
+╷
+│ Error: Delete Error
+│ 
+│ Unable to call CancelImport, got error: [DELETE /api/internal/projects/{project_id}/clusters/{cluster_id}/imports/{id}][500] CancelImport default  &{Code:59900104 Details:[] Message:failed to cancel
+│ import}
+╵
+```
+
+Cancel an import task whose status is `IMPORTING`:
+
+```
+$ terraform destroy
+...
+Plan: 0 to add, 0 to change, 1 to destroy.
+
+Do you really want to destroy all resources?
+  Terraform will destroy all your managed infrastructure, as shown above.
+  There is no undo. Only 'yes' will be accepted to confirm.
+
+  Enter a value: yes
+  
+tidbcloud_import.example_local: Destroying... [id=781074]
+tidbcloud_import.example_local: Destruction complete after 0s
+
+Destroy complete! Resources: 1 destroyed.  
+```
+
+## Configurations
+
+See [configuration documentation](https://registry.terraform.io/providers/tidbcloud/tidbcloud/latest/docs/resources/import) to get all the available configurations for the import resource.
```

Comment:
https://registry.terraform.io/providers/tidbcloud/tidbcloud/latest/docs/resources/import returns 404 Not Found.

Reply:
successfully access for me. Maybe the problem of network?
![image](https://user-images.githubusercontent.com/52435083/220239475-9514d01d-9f23-43a3-b0a1-5c2fd1c5070a.png)

Reply:
yes, I think so.